### PR TITLE
Task#7 - Feat: Price filter popover component

### LINKED ISSUE #56: Task#7 - DevTask: Create PriceFilterPopover components
## OVERVIEW
- Create manufactureFilterPopover
- Build UI for filter popover in manuFacturepopover

## DEMO
image

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Closes #56 
## OVERVIEW
- Create ManuFfactureFilterPopover
- Testing UI

## DEMO 

https://github.com/Agility-Internship/Duong-The-Hien-ReactJS/assets/113186169/60a8b02b-87e6-4746-9052-209ff4462844

### CODE REVIEW

File: practice-two/src/components/PircesFilterPopover.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * The popover automatically closes when clicking outside of it.
+ *
+ * @returns {JSX.Element} The PricePopover Component
+ */
+const PricePopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const popoverRef = useRef(null);
+
+  // Handle clicks outside the popover to close it
+  const handleClickOutside = (event) => {
```
Comment (phattruong-agilityio) [inline practice-two/src/components/PircesFilterPopover.jsx]: do you see this one is duplicate?

File: practice-two/src/components/PircesFilterPopover.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    if (popoverRef.current && !popoverRef.current.contains(event.target)) {
+      setIsPopoverOpen(false);
+    }
+  };
+
+  useEffect(() => {
```
Comment (phattruong-agilityio) [inline practice-two/src/components/PircesFilterPopover.jsx]: duplicate here

Comment (TheHienDuong) [reply practice-two/src/components/PircesFilterPopover.jsx]: Yes, Fixed and updated by adding even handle useClickOutside into components duplicated even

File: practice-two/src/components/PircesFilterPopover.jsx (hunk 1 above)
Comment (TheHienDuong) [reply practice-two/src/components/PircesFilterPopover.jsx]: Yes, I'm separated and processed useClickOutside in another file even

**Pull request merged**